tcms/testruns/templates/testruns/mutable.html

```diff
@@ -95,6 +95,59 @@
                 {{ form.build.errors }}
             </div>
         </div>
+        <div class="form-group">
+            <label class="col-md-1 col-lg-1" for="id_start_date">{% trans "Start date" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <div class="input-group date-time-picker-pf">
+                    <input type="text" class="form-control" id="id_start_date">
+                    <span class="input-group-addon">
+                        <span class="fa fa-calendar"></span>
+                    </span>
+                </div>
+
+                {% include "include/datetimepicker_script.html" with selector="#id_start_date" %}
+            </div>
+
+            <label class="col-md-1 col-lg-1" for="id_stop_date">{% trans "Stop date" %}</label>
+            <div class="col-md-3 col-lg-3">
+                <div class="input-group date-time-picker-pf">
+                    <input type="text" class="form-control" id="id_stop_date">
+                    <span class="input-group-addon">
+                        <span class="fa fa-calendar">
+                        </span>
+                    </span>
+                </div>
+
+                {% include "include/datetimepicker_script.html" with selector="#id_stop_date" %}
+            </div>
+        </div>
```

Comment:
Note to self: currently start_date is set automatically but that doesn't need to always be the case. An argument can be made for a manager creating test runs for future execution, distributing them among the team and setting planned_start/planned_stop. But we still want the actual start/stop timestamps to reflect when the work was performed, rather than setting them arbitrary. 

I can think of 2 cases:

1) If start_date is None then set that automatically upon receiving any status on any test execution
2) Convert the Started/Finished toggle button to a 3 state button to explicitly have the start reflected.